src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1042,4 +1042,7 @@
   <data name="LocalDBNotSupported" xml:space="preserve">
     <value>LocalDB is not supported on this Platform.</value>
   </data>
+  <data name="PlatformNotSupported_DataSqlClient" xml:space="preserve">
+    <value>System.Data.SqlClient is not supported on this platform.</value>
+  </data>
```

Comment:
Yes: what @joperezr said.

We have a rule with **all** packages that expose a `netstandardX.Y` reference, they **must** have a `netstandardX.Y` implementation, even if that implementation throws platformnotsupported.  This ensures that we'll never run into FileNotFound at runtime, and ensures that folks can do lightup around it without having to worry about JIT in-lining triggering a filenotfound.